src/OpenTelemetry.Exporter.Geneva/Metrics/OtlpProtobuf/OtlpProtobufSerializerHelper.cs

```diff
@@ -0,0 +1,233 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Buffers.Binary;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace OpenTelemetry.Exporter.Geneva;
+
+internal static class OtlpProtobufSerializerHelper
```

Comment:
`ProtobufSerializer` would be a better name for this class as it does not have an Otlp specific code.